main.rb

```diff
@@ -0,0 +1,45 @@
+require './app'
+def main
+  books = []
+  rental_details = []
+  people = []
+
+  options(books, people, rental_details)
+end
+
+def display_all
+  puts 'Welcome to the school library'
+  puts ' Please choose a task  basing on the number '
+  puts " 1:  Show all books.
+           2:  Show all people.
+           3: Create a person
+           4: Create a rental
+           5: Create a book
+           6: List rented books to a person by ID
+           7: Exit"
+end
+
+def options(books, people, rental_details)
+  loop do
+    display_all
+    option = gets.chomp.to_i
+    case option
+    when 1
+      show_books(books)
+    when 2
+      show_people(people)
+    when 3
+      add_people(people)
+    when 4
+      add_rental(books, people, rental_details)
+    when 5
+      add_book(books)
+    when 6
+      show_rental_details(rentals)
+    else
```

Comment:
- [ ] I'm not able to list the rentals due to the error below.  The local variable `rentals` is undefined. I think you mean `rental_details`. Check out line 4 in the `main.rb` file.

<img width="591" alt="image" src="https://user-images.githubusercontent.com/77190203/183055866-8db0b1eb-fa72-4479-8f6b-5dbfd6f2e002.png">